### PR TITLE
fix innodb status

### PR DESCRIPTION
**Pull request by bohanjason**

### CODE REVIEW

**Review by ruogudu — APPROVED** [review]

File: driver/collector/mysql_collector.py [hunk 1]
```diff
@@ -182,9 +182,10 @@ def collect_metrics(self) -> Dict[str, Any]:
         metrics["global"]["innodb_metrics"] = dict(
             self._cmd(self.METRICS_INNODB_SQL)[0]
         )
-        self._innodb_status = self._truncate_innodb_status(
-                                self._cmd(self.ENGINE_INNODB_SQL)[0][0][-1]
-                              )
+        status_raw = self._cmd(self.ENGINE_INNODB_SQL)[0]
+        if len(status_raw) > 0:
+            self._innodb_status = self._truncate_innodb_status(status_raw[0][-1])
```
Comment (NealKaviratna) [inline driver/collector/mysql_collector.py:187]: Why is the relevant portion of the raw status the first index, last index? The code here is a little hard to follow with the magic number indexes prior to being passed into the function. maybe something to clean up in the future?
```
 ? = status_raw[0]
?? = ?[-1]
self._truncate_innodb_status(??)
```
